### PR TITLE
Add datetime module to cnex

### PR DESCRIPTION
**Pull request by gitlarryf**

### CODE REVIEW

File: exec/cnex/lib/datetime.c [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+#include "number.h"
+#include "stack.h"
+
+
+
+void datetime_gmtime(TExecutor *exec)
+{
+    Number t = top(exec->stack)->number;
+
+    time_t x = number_to_sint32(t);
```
Comment (ghewgill) [inline exec/cnex/lib/datetime.c]: 32 looks wrong here. Change it to 64, I've just made the corresponding change in datetime.cpp.

File: exec/cnex/lib/datetime.c [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    n.number = number_from_uint32(tm->tm_sec);  cell_arrayAppendElement(r, n);
+    n.number = number_from_uint32(tm->tm_min);  cell_arrayAppendElement(r, n);
+    n.number = number_from_uint32(tm->tm_hour); cell_arrayAppendElement(r, n);
+    n.number = number_from_uint32(tm->tm_mday); cell_arrayAppendElement(r, n);
+    n.number = number_from_uint32(tm->tm_mon);  cell_arrayAppendElement(r, n);
+    n.number = number_from_uint32(tm->tm_year); cell_arrayAppendElement(r, n);
+    n.number = number_from_uint32(tm->tm_wday); cell_arrayAppendElement(r, n);
+    n.number = number_from_uint32(tm->tm_yday); cell_arrayAppendElement(r, n);
+    n.number = number_from_uint32(tm->tm_isdst);cell_arrayAppendElement(r, n);
+    n.number = number_from_uint32(tm->tm_sec);  cell_arrayAppendElement(r, n);
```
Comment (ghewgill) [inline exec/cnex/lib/datetime.c]: There seem to be one too many elements appended to this array here.

File: exec/cnex/lib/datetime.c (hunk 1 above)
Comment (gitlarryf) [reply exec/cnex/lib/datetime.c]: Done.

File: exec/cnex/lib/datetime.c (hunk 2 above)
Comment (gitlarryf) [reply exec/cnex/lib/datetime.c]: Opps.  Fixed.

**Pull request merged**